lib/aws/active_job/sqs/configuration.rb

```diff
@@ -17,20 +63,34 @@ class Configuration
           excluded_deduplication_keys: ['job_id']
         }.freeze
 
-        # @api private
-        attr_accessor :queues, :max_messages, :visibility_timeout,
-                      :shutdown_timeout, :client, :logger,
-                      :async_queue_error_handler, :message_group_id
+        GLOBAL_ENV_CONFIGS = %i[
+          config_file
+          threads
+          backpressure
+          max_messages
+          shutdown_timeout
+          visibility_timeout
+          message_group_id
+        ].freeze
 
-        attr_reader :excluded_deduplication_keys
+        QUEUE_ENV_CONFIGS = %i[
+          url
+          max_messages
+          visibility_timeout
+          message_group_id
+        ].freeze
+
+        QUEUE_CONFIGS = QUEUE_ENV_CONFIGS + %i[excluded_deduplication_keys]
 
         # Don't use this method directly: Configuration is a singleton class, use
-        # +Aws::ActiveJob::SQS.config+ to access the singleton config.
+        # {}Aws::ActiveJob::SQS.config} to access the singleton config.
```

Comment:
Typo. But also I think syntax is class#method. https://rubydoc.info/gems/yard/file/docs/GettingStarted.md#linking-objects

Reply:
It needs to be `.` for class methods